### DISCUSSION TITLE
Context Propagation on Lambda

### DISCUSSION THREAD

**Discussion by Vyshak1510** [open-telemetry/opentelemetry-lambda#1159] [Q&A]
I'm trying to understand if context propagation works if I have an application outside the lambda layer.
example - if i have a nodejs lambda function instrumented with opentelemetry and lets say that calls a java application hosted on ec2 also instrumented with opentelemetry, will the trace capture the entire request?

**Comment by todurnatesh** [comment]
I can confirm that it works for nodejs. But tbh it should work since its language agnostic provided the right set of instrumentation libs are available & used. In your context, both the Nodejs Lambda & java application (ec2) should enable relevant Http instrumentation lib considering the call will be over HTTP.
